YubiKit/YubiKit/Connections/Shared/Errors/YKFInvalidPinError.h

```diff
@@ -0,0 +1,22 @@
+//
+//  YKFInvalidPinError.h
+//  YubiKit
+//
+//  Created by Jens Utbult on 2024-06-19.
+//  Copyright © 2024 Yubico. All rights reserved.
+//
```

Comment:
I saw in other files you update this to a different header, maybe you forgot here?